src/molecules/Product.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { Box } from 'atomic-layout'
+import { Text } from '../atoms/Text'
+import { ProductButton } from '../atoms/Button'
+
+const StyledContainerProduct = styled.div`
+  max-width: 500px;
+  height: auto;
+`
+const StyledProductDescription = styled.div`
+  text-align: center;
+  padding: 25px 0;
```

Comment:
Well, technically, if you wish to split spacing and other visual styling you should move `padding: 25px 0` from here to the `<Box as={StyledProductDescription}>`:

```jsx
<Box as={StyledProductDescription} paddingVertical={25 / 16}>
```

P.S. This is not a serious mistake, feel free to ignore.

Reply:
I like it more.